src/modules/addObj.js

```diff
@@ -11,6 +11,7 @@ const addObj = (value) => {
   mainArr.push(object);
   storage();
   addHtml();
+  window.location.reload();
 };
 
 export default addObj;
```

Comment:
- [ ] Kindly make sure your app is working as expected. Right now it cannot add any task in the `local storage` and even display them to the UI. 
![todo](https://user-images.githubusercontent.com/83042742/192865363-4c7ef746-61c1-4e8e-b058-10223d81dcf7.gif)

Reply:
Hi [@tresorsawasawa](https://github.com/tresorsawasawa) If you click on the send button instead of 'enter' the app work as expected. This is how the template works

Reply:
Hi @IngDavidTM

I have tried both but it was the same result. 

Can you add a video like that showing that it is working and submit again. 

So the next reviewer will take that approach and check at his side if it is working.

Reply:
You're right, I just see the problem. Thank you [@tresorsawasawa](https://github.com/tresorsawasawa)